frontend/src/hooks/useUsers.js

```diff
@@ -61,8 +52,8 @@ const useUsers = (searchKeyword, sortKey, isSortAscending) => {
       listOfAllUsers
         ?.sort((a, b) => sortColumn(a, b))
         .filter((user) =>
-          makeWordComparable(user.full_name).includes(
-            makeWordComparable(searchKeyword)
+          removeAccentsFromWords(user.full_name).includes(
+            removeAccentsFromWords(searchKeyword)
```

Comment:
itt a searchKeyword-re is meghívtad a removeAccentsFromWords függvényt... 
A useProducts()-ba viszont nem, mi indokolja hogy ide kell? 

Legyen konzisztens, szóval vagy legyen mindkét helyen vagy egyiken se.

Reply:
A useUsers-ben mindkettőt közös nevezőre kell hozni, míg a products ListElement komponensében a highlighter-ben adjuk meg a függvényt, ezért a useProductsban valóban nem kellett volna már meghívni újra. Töröltem a useProducts-ból.